src/test/run-pass/tcp-stress.rs

```diff
@@ -9,6 +9,7 @@
 // except according to those terms.
 
 // ignore-android needs extra network permissions
+// TODO This comment is weird as openbsd is not mentioned in the file
```

Comment:
I was making a point about the comment above it. It's not obvious to me how openbsd is relevant.